src/components/Modals/FiatRamps/fiatRampProviders/banxa.ts

```diff
@@ -0,0 +1,36 @@
+import { adapters } from '@shapeshiftoss/caip'
+
+import { FiatRampAction, FiatRampAsset } from '../FiatRampsCommon'
+
+export const getBanxaAssets = () => {
+  const banxaAssets = adapters.getSupportedBanxaAssets()
+  const assets: FiatRampAsset[] = banxaAssets.map(asset => ({
+    assetId: asset.CAIP19,
+    symbol: asset.ticker,
+    // name will be set in useFiatRampCurrencyList hook
+    name: '',
+  }))
+  return assets
+}
+
+export const createBanxaUrl = (action: FiatRampAction, asset: string, address: string): string => {
+  const BANXA_BASE_URL = new URL('https://shapeshift.banxa.com/')
+
+  const params = new URLSearchParams()
+  params.set('fiatType', 'USD')
```

Comment:
Can we add a comment that this is not currently not supported and will default to whichever default Banxa has (AUD currently) until they implement it?

Reply:
sure